### DISCUSSION TITLE
Cant install gazebo in pop os 20.04

### DISCUSSION THREAD

**Discussion by ZA0068** [pop-os/pop#1953] [General]
I have a university project about robot simulation, and this requires Gazebo to run.

unfortunately, I can't run the one-liner script from Gazebo webpage where we install in ubuntu.

any help

**Comment by jacobgkau** [comment]
This repository is for the window tiling extension, Pop Shell. It doesn't sound like the script you're referring to would have anything to do with this tiling extension.

If you own System76 hardware, you can open a [support ticket](https://support.system76.com) for Pop!_OS support. Otherwise, you are welcome to join the [Mattermost chat room](https://chat.pop-os.org/) or [subreddit](https://old.reddit.com/r/pop_os/) for community discussion. There are also unofficial forums at [Pop!_Planet](https://pop-planet.info/). The correct GitHub repository for general Pop!_OS issues would be https://github.com/pop-os/pop.

**Reply by jacobgkau** [reply]
I saw that I can transfer discussions now, so I've transferred this one to the general Pop!_OS repository.
